doc/toolbox.1.md

```diff
@@ -40,6 +40,28 @@ access to the user's home directory, the Wayland and X11 sockets, networking
 (including Avahi), removable devices (like USB sticks), systemd journal, SSH
 agent, D-Bus, ulimits, /dev and the udev database, etc..
 
+## Supported operating system distributions
+
+By default, Toolbox tries to use an image matching the host operating system
+distribution for creating containers. If the host is not supported, then it
+fallbacks to a Fedora image.
+
+Some commands accept the `--distro` and `--release` flags allowing the user to
+use non-default distribtions and their versions. The `--distro` flag specifies
+the name of a distribution, and `--release` specifes its release version. Each
+distribution uses sligthly different versioning schema and some may not have
+any at all.
```

Comment:
> I suppose you were thinking of rolling distributions...

Yes.

> ...but we currently don't support any distribution that doesn't have a version number. So this part can be confusing.

Fair enough.

> And since we are going to list out the exact combinations that are supported, it would be clarified to the reader anyway.

:+1: